src/client/app/common/scripts/2fa.ts

```diff
@@ -0,0 +1,5 @@
+export function hexifyAB(buffer) {
+	return Array.from(new Uint8Array(buffer))
+		.map(item => item.toString(16).padStart(2, 0))
+		.join('');
+}
```

Comment:
Please insert newline at end of file.

```suggestion
}
```